examples/suite/basic_performance_quick.rb

```diff
@@ -7,12 +7,18 @@
 #  {:name => "simple json out", :config => "config/simple_json_out.conf", :input => "input/simple_10.txt", :events => 50000},
 #]
 #
+TIME = 30
+CONFIG_PATH = "examples/config/"
+INPUT_PATH = "examples/input/"
+
 [
-  {:name => "simple line in/out", :config => "config/simple.conf", :input => "input/simple_10.txt", :time => 30},
-  {:name => "simple line in/json out", :config => "config/simple_json_out.conf", :input => "input/simple_10.txt", :time => 30},
-  {:name => "json codec in/out", :config => "config/json_inout_codec.conf", :input => "input/json_medium.txt", :time => 30},
-  {:name => "line in/json filter/json out", :config => "config/json_inout_filter.conf", :input => "input/json_medium.txt", :time => 30},
-  {:name => "apache in/json out", :config => "config/simple.conf", :input => "input/apache_log.txt", :time => 30},
-  {:name => "apache in/grok codec/json out", :config => "config/simple_grok.conf", :input => "input/apache_log.txt", :time => 30},
-  {:name => "syslog in/json out", :config => "config/complex_syslog.conf", :input => "input/syslog_acl_10.txt", :time => 30},
-]
+  {:name => "simple line in/out", :config => "simple.conf", :input => "simple_10.txt", :time => TIME},
+  {:name => "simple line in/json out", :config => "simple_json_out.conf", :input => "simple_10.txt", :time => TIME},
+  {:name => "json codec in/out", :config => "json_inout_codec.conf", :input => "json_medium.txt", :time => TIME},
+  {:name => "line in/json filter/json out", :config => "json_inout_filter.conf", :input => "json_medium.txt", :time => TIME},
+  {:name => "apache in/json out", :config =>  "simple.conf", :input => "apache_log.txt", :time => TIME},
+  {:name => "apache in/grok codec/json out", :config => "simple_grok.conf", :input => "apache_log.txt", :time => TIME},
+  {:name => "syslog in/json out", :config => "complex_syslog.conf", :input => "syslog_acl_10.txt", :time => TIME},
+].map do |test|
+  test.merge({:config => CONFIG_PATH + test[:config], :input => INPUT_PATH + test[:input], :time => TIME})
```

Comment:
oh, I did not even see the `lib/lsperfm/defaults` ... this is weird. why have both? this is uselessly duplicated. I think it should all be put into the examples dir and be runnable from there?